kernel/aster-nix/src/syscall/prctl.rs

```diff
@@ -35,6 +35,14 @@ pub fn sys_prctl(option: i32, arg2: u64, arg3: u64, arg4: u64, arg5: u64) -> Res
 
             write_val_to_user(write_to_addr, &write_val)?;
         }
+        PrctlCmd::PR_GET_DUMPABLE => todo!(),
```

Comment:
`todo!()` will leads to a kernel panic here,  If we do not support it ,just leave it in  `_ => todo!(),`